lane.py

```diff
@@ -1923,18 +1940,19 @@ def works_from_database(self, _db, facets=None, pagination=None, **kwargs):
         lanes can be generated through search engine queries.
 
         :param _db: A database connection.
-        :param facets: A DatabaseBackedFacets object which may put additional
-           constraints on WorkList membership.
+        :param facets: A Facets object which may place additional
+           constraints on WorkList membership. Other objects are ignored.
```

Comment:
I see what you mean by this based on the code, but it would be clearer as "Faceting objects are ignored unless they are an instance of Facets or one of its subclasses."

Reply:
Or, if you make the other changes, add a `:raises ValueError` to describe this situation rather than changing the description of `facets`.